faker/providers/passport/ru_RU/__init__.py

```diff
@@ -0,0 +1,9 @@
+from typing import Tuple
+
+from .. import Provider as PassportProvider
+
+
+class Provider(PassportProvider):
+    def passport_owner(self, gender: str = "M") -> Tuple[str, str]:
```

Comment:
you can use `faker.typing.SexLiteral` as type for the `gender` argument.

Reply:
Agree, let I will fix it! 

Thanks for feedback.